src/redux/actions/LoginActions.js

```diff
@@ -0,0 +1,23 @@
+import { toast } from 'react-toastify';
+import axios from 'axios';
+import { loginTypes } from './types';
+
+
+export const loginAction = (userData) => dispatch => {
+  return axios.post('https://inviticus-staging.herokuapp.com/api/users/login/', userData)
```

Comment:
Place these URLs in a file and import them. For consistency, reusavility and to avoid typo errors.